canvas/biplot.py

```diff
@@ -72,23 +111,29 @@ def make_biplot(samples,
             arrow_head = value
         if key == 'feat_leg_loc':
             feat_leg_loc = value
+        if key == 'feat_leg_ncol':
+            feat_leg_ncol = value
+        if key == 'feat_leg_ls':
+            feat_leg_ls = value
         if key == 'feature_order':
             if value == 0:
                 feature_order = 0
             if value == 1:
                 feature_order = 1
-        if key == 'sample_drop_list':
-            sample_drop_list = value
+        if key == 'samp_drop_list':
+            samp_drop_list = value
         if key == 'show_color_drop':
             show_color_drop = value
-        if key == 'sample_drop_col':
-            sample_drop_col = value
+        if key == 'samp_drop_col':
+            samp_drop_col = value
         if key == 'eigenvalues':
             eigenvalues = value
         if key == 'x_pad':
             x_pad = value
         if key == 'y_pad':
             y_pad = value
+        if key == 'show_feat_legend':
+            show_feat_legend = value
 
```

Comment:
I'm looking at all of these parameters.  They are not standard matplotlib parameters.  So we need to add more documentation to determine exactly what they are used for.